pkg/service/manipulator.go

```diff
@@ -82,6 +83,11 @@ func (m *manipulator) Process(spec ProcessSpec) ([]byte, error) {
 		data = m.processor.GrayScale(data)
 		trackDuration(grayScaleDurationKey, t, spec)
 	}
+	if CleanFloat(params[blur], 1000) > 0 {
```

Comment:
can be improved so that we don't need to call this twice:
`if radius := CleanFloat(params[blur], 1000); radius > 0`

Reply:
also we can increase this to 2000 I think, imgix supports blurring until up to 2 thousands (https://docs.imgix.com/apis/url/stylize/blur)

Reply:
IMHO, for CPU based processing their should be a lower upper bound because only a blur value of 50-60 takes up too much time to process. The supported blur value can be function of input image size since a blur of value 5 on a 2160p image takes longer than a blur of value 10 on a 720p image.

Reply:
Let's keep it 1000 for now because I feel we will not see much difference between 1000 and 2000 at most given images.